contracts/bancor-portal/BancorPortal.sol

```diff
@@ -367,4 +372,34 @@ contract BancorPortal is IBancorPortal, ReentrancyGuardUpgradeable, Utils, Upgra
             );
         }
     }
+
+    /**
+     * @dev replaces native token with WETH
+     */
+    function _nativeToWeth(Token token) private view returns (Token) {
+        if (token.isNative()) {
+            return Token(address(_weth));
+        }
+        return token;
+    }
+
+    /**
+     * @dev replaces WETH with the native token
+     */
+    function _wethToNative(Token token) private view returns (Token) {
+        if (_isWETH(token)) {
```

Comment:
Function `_isWETH` seems redundant by now; you may as well do `if (address(token) == address(_weth))` here.